### DISCUSSION TITLE
The object type returned by any CTkinter widget is of NoneType therefore unable to use get() method for fetching values

### DISCUSSION THREAD

**Discussion by aryant-nigam** [TomSchimansky/CustomTkinter#350] [General]
CTkinter widget constructors are not returning an object of respective widget type instead returning NoneType object that turns out to be of no use

variable_name_entry = customtkinter.CTkEntry(master=new_entry_row_frame).grid(row=0,column=0,padx=15,pady=5)
text = variable_name_entry .get()

err: NoneType Object has no method named get()

**Comment by TomSchimansky** [comment]
customtkinter.CTkEntry() is returning the object but .grid() does not! You need to call the grid method on the object after you created it:
```python
variable_name_entry = customtkinter.CTkEntry(master=new_entry_row_frame)
variable_name_entry.grid(row=0, column=0, padx=15, pady=5)
text = variable_name_entry.get()
```
